drivers/clock_control/clock_stm32_ll_u5.c

```diff
@@ -114,6 +114,19 @@ static uint32_t get_pllsrc_frequency(void)
 	return -EIO;
 }
 
+static uint32_t get_startup_frequency(void)
+{
+	switch (LL_RCC_GetSysClkSource()) {
+	case LL_RCC_SYS_CLKSOURCE_STATUS_MSIS:
+		return get_msis_frequency();
+	case LL_RCC_SYS_CLKSOURCE_STATUS_HSI:
+		return STM32_HSI_FREQ;
+	default:
+		__ASSERT(0, "Unexpected startup freq");
+		return -EIO;
```

Comment:
function `get_startup_frequency` is returning **uint32**

Reply:
fixed